### ISSUE TITLE
v2.41 breaks macOS support

### ISSUE THREAD

**Issue by sladewatkins** [Xpl0itU/WiiUDownloader#89]
Just an FYI, on ``Mac14,7`` (MacBook Pro 2022, Apple M2, 16GB RAM) running macOS 14.4.1 (Sonoma), version 2.41's dependencies and go updates completely broke macOS support. Reverting to version 2.40 fixes the problem.

**Comment by Xpl0itU** [comment]
It works for me just fine, what exactly happens?

**Comment by sladewatkins** [comment]
It opens, and then immediately crashes, that's literally it. There's no way to fix it except for downgrading to version v2.40, which solves the problem. Not sure what's going on here...

**Comment by Xpl0itU** [comment]
Hmm, do you have homebrew installed in your mac? If so, try running `brew install gtk+3` and run WiiUDownloader again and see if it crashes

**Comment by sladewatkins** [comment]
Okay, interesting. ``gtk+3`` was already installed, so I just removed it and installed it again. No problems now as far as I can tell. 

For those who may run into this from Google or something, here's how I fixed it:
- First off, make sure you have Homebrew ([brew.sh](https://brew.sh)).
- ``brew uninstall gtk+3`` (if you know you don't have ``gtk+3`` then skip to installing it)
- ``brew install gtk+3``
- Update WiiUDownloader to or install the latest version, should launch fine.

Sorry for the noise, @Xpl0itU. Appreciate you pointing that out though, I didn't even think of this.
